speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/ResearcherMenu.cpp

```diff
@@ -231,7 +410,16 @@ void* ResearcherMenuInit(void* p_nextMenu)
                                    NULL, (tfuiCallback)NULL, 1);
     s_nextHandle = p_nextMenu;
 
+    // Retrieves the saved user xml file, if it doesn't exist it retrieves the default xml file
+    std::string strPath("config/ResearcherMenu.xml");
+    char buf[512];
+    sprintf(buf, "%s%s", GfLocalDir(), strPath.c_str());
     void* param = GfuiMenuLoad("ResearcherMenu.xml");
+    if (GfFileExists(buf)) {
+        param = GfParmReadFile(buf, GFPARM_RMODE_STD);
+    }
+    // Initialize settings with the retrieved xml file
+    InitializeSettings(param);
```

Comment:
Somewhere it goes wrong in Complete Takeover (it is always on when initializing)